### ISSUE TITLE
Class - Hunter (inline - Pet)

### ISSUE THREAD

**Issue by Skylark14** [scotty0100/wowascension-issues#98]
Name: Breena
Account Name (for vote points rewards): Skylark
Map & Area: n/a
How to replicate the bug: not sure 
What happens: Pet is stuck at one bar on level 33, I am now level 37.  He was keeping pace till around level 35 but seems to have gained no xp since I gained 2 levels.
What should happen: He should level up at roughly the same rate.
Located on WoWHead Map (if applicable):
Source (proof, if applicable): 

Extra Information:

**Comment by Deviastarix** [comment]
Bump!
I have the same issue. I've gained almost a full level, and my pet is stuck on 0 XP for that level. Both my character and pet are on level 12.

**Comment by Faraday1** [comment]
The pet will not gain xp until you are one level above it, Devia. It needs a lot less xp so it catches up rather quickly until it is the same level as you.

**Comment by Deviastarix** [comment]
Yep, thanks. I eventually noticed that. I don't remember it being that way on other hunters I've levelled, and thought they should gain a percentage of the XP I gain as I go.
Maybe the fact that it's classed as a Minion here, rather than a Pet has something to do with that.
Anyway, it's not a problem now that I don't have to put her down. :)

**Comment by kamicolo** [comment]
https://github.com/scotty0100/wowascension-issues/issues/49
